### ISSUE TITLE
Rule 6-3 Error with Arguments passed to `std_equal_with_precision`

### ISSUE THREAD

**Issue by JacksonJ-KC** [pnnl/ruleset-checking-tool#1587]
rct229\rulesets\ashrae9012019\section6\section6rule3.py, line 105, in rule_check
    return self.precision_comparison["total_space_lpd_p"](
  File "C:\Users\JacksonJarboe\Documents\Development\RCT\rct229\utils\std_comparisons.py", line 74, in std_equal_with_precision
    raise TypeError(
TypeError: Arguments must be consistent in type: all Quantity or all non-Quantity.

**Comment by JacksonJ-KC** [comment]
`val` is `int`
`std_val` is `int`
`precision` is `Quantity(0.01, 'watt / square foot')`

**Comment by JacksonJ-KC** [comment]
```
                total_space_lpd_u = sum(
                    find_all("$.interior_lighting[*].power_per_area", space_u)
                ) or ZERO.POWER_PER_AREA
                total_space_lpd_p = sum(
                    find_all("$.interior_lighting[*].power_per_area", space_p)
                ) or ZERO.POWER_PER_AREA
```

**Comment by yunjoonjung-PNNL** [comment]
Addressed in PR #1589
